cmd/pull.go

```diff
@@ -40,15 +40,20 @@ var pullCmd = &cobra.Command{
 			targetDir = "skuidpages"
 		}
 
-		//build the module query paramater
-		var requestedModule = []string{""}
+		//build the module and name query paramaters
+		//var requestedModule = []string{""}
+		var requestedItems = []string{""}
+		var itemKey = "module"
 
 		if module != "" {
-			requestedModule = []string{module}
+			requestedItems = []string{module}
+		} else if name != "" {
+			requestedItems = []string{name}
+			itemKey = "name"
 		}
 
 		query := url.Values{
-			"module": requestedModule,
+			itemKey: requestedItems,
 		}
 
 		//query the API for all pages in the requested module
```

Comment:
i think it might be more idiomatic to write this chunk of code like so: 

```
query := url.Values{}
if module != {
  query.Add("module", []string{module})
} else if name != "" {
  query.Add("name", []string{name})
}
```